communitybridge/2020/q3-q4/selected_projects.md

```diff
@@ -63,9 +65,38 @@ Project maintainers and mentors, please submit the selected below (under the Sel
 	-	https://github.com/kubernetes-csi/csi-driver-smb/issues
 -	Community Bridge project (URL): https://people.communitybridge.org/project/2d438b9a-c539-46d0-9eed-c6ee4404c88a
 
-### Prometheus
+#### Thanos
+
+##### Receive: Hashring Update Improvements
+
+- Description: urrently, any change to the hashring configuration file will trigger all Thanos Receive nodes to flush their multi-TSDBs, causing them to enter an unready state until the flush is complete. This unavailability during a flush allows for a clear state transition, however it can result in downtimes on the order of five minutes for every configuration change. We propose modifying how the Thanos Receive component re-configures itself after the hashring configuration file has changed so that the system experiences no downtime.
```

Comment:
#
```suggestion
- Description: Currently, any change to the hashring configuration file will trigger all Thanos Receive nodes to flush their multi-TSDBs, causing them to enter an unready state until the flush is complete. This unavailability during a flush allows for a clear state transition, however it can result in downtimes on the order of five minutes for every configuration change. We propose modifying how the Thanos Receive component re-configures itself after the hashring configuration file has changed so that the system experiences no downtime.
```